plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPluginTest.kt

```diff
@@ -599,15 +565,16 @@ class KtlintPluginTest : AbstractPluginTest() {
     fun forceDependenciesRuleSetConfiguration(gradleVersion: GradleVersion) {
         project(gradleVersion) {
             withCleanSources()
-
             //language=Groovy
             buildGradle.appendText(
                 """
-
-                dependencies {
-                    $KTLINT_RULESET_CONFIGURATION_NAME "com.pinterest.ktlint:ktlint-cli-ruleset-core:1.0.0"
-                }
-                """.trimIndent()
+ktlint {
+    version = "1.0.1"
+}
+dependencies {
+    $KTLINT_RULESET_CONFIGURATION_NAME "com.pinterest.ktlint:ktlint-cli-ruleset-core:1.0.0"
+}
+"""
```

Comment:
Formatting?

Reply:
this is another valid way of formatting text blocks. I prefer it as the other way tends to seem like it has an arbitrary indentation level, and this way does not require a trim function